docs/versioned_docs/version-zenith/user-guide/cloud/572923-get-started.mdx

```diff
@@ -0,0 +1,314 @@
+
+    ![Get token](/img/versioned_docs/version-zenith/user-guide/cloud/get-started/get-token.png)
+
+  :::warning
+  If you regenerate your token, you must replace the token wherever you've referenced it. To reduce operational interruptions, only regenerate your token if it has been leaked.
+  :::
+
+Once you have your token, you can now use it to connect Dagger Cloud with your CI and/or local development machines.
+
+### Connect from your local development host
+
+You can visualize and debug your local Dagger pipeline runs with Dagger Cloud to identify issues before pushing them to CI.
+
+To configure for local development, set your Dagger Cloud token as an environment variable. For example:
+
+```shell
+export DAGGER_CLOUD_TOKEN={your token}
+```
+
+:::info BEST PRACTICES FOR VISUALIZING LOCAL RUNS IN DAGGER CLOUD
+Dagger Cloud uses metadata from your version control system (VCS), such as Git, to populate some information in the user interface. Therefore, you should run your Dagger pipelines within a VCS context, such as a local directory linked to a cloned Git repository.
+:::
+
+### Connect from your CI engine
+
+The general procedure to connect Dagger Cloud with a CI provider/CI tool is:
+
+- Add your Dagger Cloud token to your CI workflows.
+  - Store the Dagger Cloud token as a secret with your CI.
+    :::danger Keep your Dagger Cloud token private
+    You must store the Dagger Cloud token as a secret (not plaintext) with your CI provider and reference it in your CI's workflow. Using a secret is recommended to protect your Dagger Cloud account from being used by forks of your project. We provide links in the steps below for configuring secrets with popular CI tools.
+    :::
+  - Add the secret to your CI environment as a variable named `DAGGER_CLOUD_TOKEN`.
+- For **Team** plan subscribers with ephemeral CI runners only: Add a wait step (example below) for the Docker configuration to allow the Dagger Engine to push cache volumes to the experimental Dagger Cloud distributed cache. Subscribers on the **Individual** plan do not have access to the experimental distributed cache and do not need to add this step.
+- If you are using GitHub Actions, install the Dagger Cloud GitHub app for GitHub Checks. This app adds a GitHub Check to your pull requests and links to the run status and pipeline visualization for the commit in Dagger Cloud.
+
+You can use Dagger Cloud whether you're hosting your own CI runners and infrastructure or using hosted/SaaS runners.
+
+:::danger
+When using self-hosted CI runners on AWS infrastructure, NAT Gateways are a common source of unexpected network charges. It's advisable to setup an Amazon S3 Gateway for these cases. Refer to the [AWS documentation](https://docs.aws.amazon.com/vpc/latest/privatelink/vpc-endpoints-s3.html) for detailed information on how to do this.
+:::
+
+<Tabs groupId="ci">
```

Comment:
IMPORTANT: I have only tested the GHA version, the rest are guesstimates. As @kpenfound has the most experience with this, I would like him to review and update as needed.